src/satellite/test.js

```diff
@@ -75,9 +75,8 @@ describe('Satellite()', () => {
     token = createToken({ sub: 'test-user@email.com' });
   });
 
-  afterEach((done) => {
-    service.stop(done);
-    service = null;
+  afterEach(async () => {
```

Comment:
You don't need async/await here, since you're not doing anything with the return value:

```js
afterEach(() => service.stop());
```